AWapi.playground/Contents.swift

```diff
@@ -0,0 +1,101 @@
+//: Playground - noun: a place where people can play
+
+import UIKit
+import PlaygroundSupport
+
+PlaygroundPage.current.needsIndefiniteExecution = true
+
+var str = "Hello, playground"
+
+import CoreData
+import Foundation
+
+let baseURL = "https://www.americanwhitewater.org/content/"
+
+let riverURL = baseURL + "River/search/.json"
+
+struct AWReachInfo: Codable {
+    //swiftlint:disable:next identifier_name
+    let id: Int //
+    let abstract: String? //
+    let avgGradient: Int16? //
+    let photoId: Int32? //
+    let length: String? //
+    let maxGradient: Int16? //
+    let description: String? //
+    let shuttleDetails: String? //
+    let zipcode: String?
+
+    enum CodingKeys: String, CodingKey {
+        //swiftlint:disable:next identifier_name
+        case id, abstract
+        case avgGradient = "avggradient"
+        case photoId = "photoid"
+        case length
+        case maxGradient = "maxgradient"
+        case description
+        case shuttleDetails = "shuttledetails"
+        case zipcode
+    }
+}
+
+struct AWGageInfo: Codable {
+    //swiftlint:disable:next identifier_name
+    let id: Int
+    //let name: String
+    //let metricid: Int
+}
+
+struct AWReachMain: Codable {
+    let info: AWReachInfo
+    let guages: [AWGageInfo]
```

Comment:
Is this misspelled in the api?

Reply:
I believe the API used all three variants of gage spelling. I standardized on gage in 
0eb5d49